action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   delta:
     description: 'Difference between the old and final test coverage'
     default: 100
+  total_delta:
+    description: 'Difference between the old and final test coverage at the total level'
+    default: null
```

Comment:
should this be 100 instead of null?

Reply:
I was thinking about that, but I left it as null because if the user wants to keep say a delta of 5 across _all_ coverage lines, they'd have to add 2 new fields
```
delta: 5
total_delta: 5
```

otherwise, it would take 5 for files and keep 100 defaulted for the total line

This way, if the total_delta is not there, just fall back to `delta` :)

Reply:
gotcha. makes sense!